tutorials/diagnostics.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    setup = MandelWithDiagnostics({"domain_size": (10, 10)})
</pre><p>Why change the model in the middle of the tutorial?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/813/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>